tests/e2e_regression/edenfast/e2e_config_1.yml

```diff
@@ -52,6 +52,10 @@ server:
   storage:
     endpoint: postgresql://api:password@localhost:5432/indexer?sslmode=disable
     backend: postgres
+  # Set some reserved addresses, so that circulating supply functionality is tested.
+  circulating_supply_reserved_addresses:
+    - oasis1qp3r8hgsnphajmfzfuaa8fhjag7e0yt35cjxq0u4
```

Comment:
are we allowed to make the actual list public

Reply:
Not sure, but these are "random" accounts I picked from the chain

Reply:
eh ok